ui/panels-plugin/src/plugins/time-series-chart/VisualOptionsEditor.tsx

```diff
@@ -99,6 +105,28 @@ export function VisualOptionsEditor({ value, onChange }: VisualOptionsEditorProp
           />
         }
       />
+      <OptionsEditorControl
+        label={VISUAL_CONFIG.stack.label}
+        control={
+          <Autocomplete
```

Comment:
why not use `ToggleButtons`? 😛

Reply:
😂  So my [first commit](https://github.com/perses/perses/pull/997/commits/c53fb85fd5b6f02d666fd1b97938c442d2d99dee#diff-217014ce88a4481e76f779f48d62dbb61b6aae3be28f42fdb7ade48afcb6ed5a) actually used ToggleButtons, since there are going to be more than 2 options (adding 'Percent') Mohit recommended switching to a dropdown so it's consistent with the other fields

![normal_stack_area_ex](https://user-images.githubusercontent.com/9369625/221925932-e71dfc5d-0f6d-4179-90c1-90625afc92c1.png)